docs/general/ad-filtering/create-own-filters.md

```diff
@@ -2218,6 +2244,16 @@ Rules with `$removeparam` modifier are supported by AdGuard for Windows, Mac and
 
 :::
 
+:::caution Restrictions
+
+AdGuard Browser Extension MV3 has some limitations:
+
+- Allowlist rules are not supported.
+- Regexp, negation and allow rules are not supported.
```

Comment:
- Allowlist rules are not supported
- allow rules are not supported

Isn't it the same?